src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0-*",
+
+    "dependencies": {
+        "Microsoft.NETCore.App": {
+            "type": "platform",
+            "version": "1.0.0-rc2-23931"
```

Comment:
The pretemplate gets its version replaced by this code:  https://github.com/dotnet/cli/pull/2331/files#diff-1308dc2ffd19c4821c162b7be1473f37R98